src/rust/lib/frp/Cargo.toml

```diff
@@ -22,8 +24,14 @@ unicode-segmentation = { version = "1.6.0"                               }
 # and this is vulnerable to any wasm-bindgen version change.
 wasm-bindgen = { version = "=0.2.58" , features = ["nightly"] }
 
+
 [dependencies.web-sys]
 version = "0.3.4"
 features = [
   'KeyboardEvent',
 ]
+
+
```

Comment:
This is the only Cargo.toml with additional spacing. And I don't think it is needed - those files are usually short, and the sections are clearly visible.